distbench.proto

```diff
@@ -218,6 +222,7 @@ message GenericRequest {
   optional bytes payload = 1;
   optional int32 rpc_index = 2;
   optional TraceContext trace_context = 3;
+  optional bool warmup = 4;
```

Comment:
Should we add  [default = false] ? It is guaranteed to default to false, but it may be more explicit for somebody unfamiliar with protos and we rely on this behavior.

Reply:
I don't think it's necessary. This field is set and used by the distbench_engine code automatically. It's an implementation detail, not part of the traffic config or even the protocol driver.